resources/src/appleMain/kotlin/dev/icerock/moko/resources/MokoResourcesBundle.kt

```diff
@@ -0,0 +1,8 @@
+package dev.icerock.moko.resources
+
+import dev.icerock.moko.resources.utils.loadableBundle
+import platform.Foundation.NSBundle
+
+object MokoResourcesBundle {
+    val bundle: NSBundle by lazy { NSBundle.loadableBundle("com.icerockdev.library.MR") }
```

Comment:
invalid change. bundle should be used from MR class generated. with this code multimodule support will be broken

Reply:
all this object not needed - MR class replace it